app/lib/nomis_client/allocations.rb

```diff
@@ -12,6 +12,16 @@ def post(booking_id:, body_params: {})
 
         e.response
       end
+
+      def put(booking_id:, event_id:, body_params: {})
```

Comment:
I see the Nomis routes and names are crazy; nevertheless `put` doesn't seem like a good name here if this method will _always_ cancel an event? if so, how about `cancel`?

Reply:
This matches the convention with the other nomis client services - i.e. the HTTP method that will be executed. I really don't want to break that convention - it delegates to `put` on the oauth token under the covers too.

Reply:
👍  fair enough